# Running a pre-trained YOLO classification model

Since we don't have bounding boxes (and we can't feasibly label all of the bounding boxes in the timelines needed for this project), we're still going to try to make use of pre-trained models and transfer learning to still improve on our model.

The hope is that since we have a relatively smaller-scale dataset (with only aorund 10,000 tornadic exampels to train on), we might see benefits from freezing early layers of a pre-trained model with a lot more data and training only the later layers. 

Ultralytics has a pre-trained YOLO classification model that might come in handy here - we'll try freezing the early layers and training on that front.

In [1]:
from ultralytics import YOLO

# Load a model
model = YOLO("yolo11n-cls.pt")  # load a pretrained model (recommended for training)

dataset_path = "/mnt/d/tornetExamples/all_2014"

# Train the model
results = model.train(data="/mnt/d/tornetExamples/all_2014", epochs=1, save=True, save_period=1, batch=4, freeze=7, imgsz=560)

Ultralytics 8.3.36 🚀 Python-3.9.20 torch-2.5.1+cu124 CUDA:0 (NVIDIA GeForce RTX 3070, 8192MiB)
engine/trainer: task=classify, mode=train, model=yolo11n-cls.pt, data=/mnt/d/tornetExamples/all_2014, epochs=1, time=None, patience=100, batch=4, imgsz=560, save=True, save_period=1, cache=False, device=None, workers=8, project=None, name=train9, exist_ok=False, pretrained=True, optimizer=auto, verbose=True, seed=0, deterministic=True, single_cls=False, rect=False, cos_lr=False, close_mosaic=10, resume=False, amp=True, fraction=1.0, profile=False, freeze=7, multi_scale=False, overlap_mask=True, mask_ratio=4, dropout=0.0, val=True, split=val, save_json=False, save_hybrid=False, conf=None, iou=0.7, max_det=300, half=False, dnn=False, plots=True, source=None, vid_stride=1, stream_buffer=False, visualize=False, augment=False, agnostic_nms=False, classes=None, retina_masks=False, embed=None, show=False, save_frames=False, save_txt=False, save_conf=False, save_crop=False, show_labels=True, show_con

E0000 00:00:1732482457.456175      38 cuda_dnn.cc:8310] Unable to register cuDNN factory: Attempting to register factory for plugin cuDNN when one has already been registered
E0000 00:00:1732482457.471680      38 cuda_blas.cc:1418] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered


Overriding model.yaml nc=80 with nc=2

                   from  n    params  module                                       arguments                     
  0                  -1  1       464  ultralytics.nn.modules.conv.Conv             [3, 16, 3, 2]                 
  1                  -1  1      4672  ultralytics.nn.modules.conv.Conv             [16, 32, 3, 2]                
  2                  -1  1      6640  ultralytics.nn.modules.block.C3k2            [32, 64, 1, False, 0.25]      
  3                  -1  1     36992  ultralytics.nn.modules.conv.Conv             [64, 64, 3, 2]                
  4                  -1  1     26080  ultralytics.nn.modules.block.C3k2            [64, 128, 1, False, 0.25]     
  5                  -1  1    147712  ultralytics.nn.modules.conv.Conv             [128, 128, 3, 2]              
  6                  -1  1     87040  ultralytics.nn.modules.block.C3k2            [128, 128, 1, True]           
  7                  -1  1    295424  ultralytics

train: Scanning /mnt/d/tornetExamples/all_2014/train... 17447 images, 0 corrupt: 100%|██████████| 17447/17447 [00:00<?, 
val: Scanning /mnt/d/tornetExamples/all_2014/test... 2546 images, 0 corrupt: 100%|██████████| 2546/2546 [00:00<?, ?it/s]


optimizer: 'optimizer=auto' found, ignoring 'lr0=0.01' and 'momentum=0.937' and determining best 'optimizer', 'lr0' and 'momentum' automatically... 
optimizer: AdamW(lr=0.000714, momentum=0.9) with parameter groups 39 weight(decay=0.0), 40 weight(decay=0.0005), 40 bias(decay=0.0)
TensorBoard: model graph visualization added ✅
Image sizes 576 train, 576 val
Using 8 dataloader workers
Logging results to runs/classify/train9
Starting training for 1 epochs...

      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:44<00:00,  7.16it/s]


                   all      0.868          1

1 epochs completed in 0.159 hours.
Optimizer stripped from runs/classify/train9/weights/last.pt, 3.2MB
Optimizer stripped from runs/classify/train9/weights/best.pt, 3.2MB

Validating runs/classify/train9/weights/best.pt...
Ultralytics 8.3.36 🚀 Python-3.9.20 torch-2.5.1+cu124 CUDA:0 (NVIDIA GeForce RTX 3070, 8192MiB)
YOLO11n-cls summary (fused): 112 layers, 1,528,586 parameters, 0 gradients, 3.2 GFLOPs
WARNING ⚠️ Dataset 'split=val' not found, using 'split=test' instead.
train: /mnt/d/tornetExamples/all_2014/train... found 17447 images in 2 classes ✅ 
val: None...
test: /mnt/d/tornetExamples/all_2014/test... found 2546 images in 2 classes ✅ 


               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:17<00:00, 18.54it/s]


                   all      0.868          1
Speed: 0.5ms preprocess, 1.1ms inference, 0.0ms loss, 0.0ms postprocess per image
Results saved to runs/classify/train9


In [3]:
# Train the model
results_2 = model.train(data="/mnt/d/tornetExamples/all_2014", epochs=2, save=True, save_period=1, batch=4, freeze=7, imgsz=560)

Ultralytics 8.3.36 🚀 Python-3.9.20 torch-2.5.1+cu124 CUDA:0 (NVIDIA GeForce RTX 3070, 8192MiB)
engine/trainer: task=classify, mode=train, model=yolo11n-cls.pt, data=/mnt/d/tornetExamples/all_2014, epochs=2, time=None, patience=100, batch=4, imgsz=560, save=True, save_period=1, cache=False, device=None, workers=8, project=None, name=train92, exist_ok=False, pretrained=True, optimizer=auto, verbose=True, seed=0, deterministic=True, single_cls=False, rect=False, cos_lr=False, close_mosaic=10, resume=False, amp=True, fraction=1.0, profile=False, freeze=7, multi_scale=False, overlap_mask=True, mask_ratio=4, dropout=0.0, val=True, split=val, save_json=False, save_hybrid=False, conf=None, iou=0.7, max_det=300, half=False, dnn=False, plots=True, source=None, vid_stride=1, stream_buffer=False, visualize=False, augment=False, agnostic_nms=False, classes=None, retina_masks=False, embed=None, show=False, save_frames=False, save_txt=False, save_conf=False, save_crop=False, show_labels=True, show_co

train: Scanning /mnt/d/tornetExamples/all_2014/train... 17447 images, 0 corrupt: 100%|██████████| 17447/17447 [00:00<?, 
val: Scanning /mnt/d/tornetExamples/all_2014/test... 2546 images, 0 corrupt: 100%|██████████| 2546/2546 [00:00<?, ?it/s]


optimizer: 'optimizer=auto' found, ignoring 'lr0=0.01' and 'momentum=0.937' and determining best 'optimizer', 'lr0' and 'momentum' automatically... 
optimizer: AdamW(lr=0.000714, momentum=0.9) with parameter groups 39 weight(decay=0.0), 40 weight(decay=0.0005), 40 bias(decay=0.0)
TensorBoard: model graph visualization added ✅
Image sizes 576 train, 576 val
Using 8 dataloader workers
Logging results to runs/classify/train92
Starting training for 2 epochs...

      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:49<00:00,  6.44it/s]

                   all      0.864          1



      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:52<00:00,  6.07it/s]

                   all      0.865          1



2 epochs completed in 0.342 hours.
Optimizer stripped from runs/classify/train92/weights/last.pt, 3.2MB
Optimizer stripped from runs/classify/train92/weights/best.pt, 3.2MB

Validating runs/classify/train92/weights/best.pt...
Ultralytics 8.3.36 🚀 Python-3.9.20 torch-2.5.1+cu124 CUDA:0 (NVIDIA GeForce RTX 3070, 8192MiB)
YOLO11n-cls summary (fused): 112 layers, 1,528,586 parameters, 0 gradients, 3.2 GFLOPs
WARNING ⚠️ Dataset 'split=val' not found, using 'split=test' instead.
train: /mnt/d/tornetExamples/all_2014/train... found 17447 images in 2 classes ✅ 
val: None...
test: /mnt/d/tornetExamples/all_2014/test... found 2546 images in 2 classes ✅ 


               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:16<00:00, 19.37it/s]


                   all      0.865          1
Speed: 0.5ms preprocess, 1.0ms inference, 0.0ms loss, 0.0ms postprocess per image
Results saved to runs/classify/train92


In [2]:
from ultralytics import YOLO

# Load a model
model = YOLO("yolo11n-cls.pt")  # load a pretrained model (recommended for training)

dataset_path = "/mnt/d/tornetExamples/all_2014"

# Train the model
results_3 = model.train(data="/mnt/d/tornetExamples/all_2014", epochs=10, save=False, save_period=1, batch=4, freeze=7, imgsz=560)

Ultralytics 8.3.36 🚀 Python-3.9.20 torch-2.5.1+cu124 CUDA:0 (NVIDIA GeForce RTX 3070, 8192MiB)
engine/trainer: task=classify, mode=train, model=yolo11n-cls.pt, data=/mnt/d/tornetExamples/all_2014, epochs=10, time=None, patience=100, batch=4, imgsz=560, save=False, save_period=1, cache=False, device=None, workers=8, project=None, name=train11, exist_ok=False, pretrained=True, optimizer=auto, verbose=True, seed=0, deterministic=True, single_cls=False, rect=False, cos_lr=False, close_mosaic=10, resume=False, amp=True, fraction=1.0, profile=False, freeze=7, multi_scale=False, overlap_mask=True, mask_ratio=4, dropout=0.0, val=True, split=val, save_json=False, save_hybrid=False, conf=None, iou=0.7, max_det=300, half=False, dnn=False, plots=True, source=None, vid_stride=1, stream_buffer=False, visualize=False, augment=False, agnostic_nms=False, classes=None, retina_masks=False, embed=None, show=False, save_frames=False, save_txt=False, save_conf=False, save_crop=False, show_labels=True, show_

train: Scanning /mnt/d/tornetExamples/all_2014/train... 17447 images, 0 corrupt: 100%|██████████| 17447/17447 [00:00<?, 
val: Scanning /mnt/d/tornetExamples/all_2014/test... 2546 images, 0 corrupt: 100%|██████████| 2546/2546 [00:00<?, ?it/s]


optimizer: 'optimizer=auto' found, ignoring 'lr0=0.01' and 'momentum=0.937' and determining best 'optimizer', 'lr0' and 'momentum' automatically... 
optimizer: AdamW(lr=0.000714, momentum=0.9) with parameter groups 39 weight(decay=0.0), 40 weight(decay=0.0005), 40 bias(decay=0.0)
TensorBoard: model graph visualization added ✅
Image sizes 576 train, 576 val
Using 8 dataloader workers
Logging results to runs/classify/train11
Starting training for 10 epochs...

      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:45<00:00,  6.98it/s]

                   all      0.861          1



      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:54<00:00,  5.83it/s]

                   all      0.862          1



      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:54<00:00,  5.81it/s]

                   all      0.872          1



      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.22it/s]

                   all      0.859          1



      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.23it/s]

                   all      0.875          1



      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:52<00:00,  6.09it/s]

                   all      0.877          1



      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.17it/s]

                   all      0.877          1



      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.17it/s]

                   all      0.886          1



      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:53<00:00,  5.97it/s]

                   all      0.881          1



      Epoch    GPU_mem       loss  Instances       Size


      10/10     0.237G      0.159          3        576: 100%|██████████| 4362/4362 [09:00<00:00,  8.07it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.21it/s]

                   all      0.876          1



10 epochs completed in 1.634 hours.
Optimizer stripped from runs/classify/train11/weights/last.pt, 3.2MB


In [1]:
from ultralytics import YOLO

# Load a model
model = YOLO("yolo11n-cls.pt")  # load a pretrained model (recommended for training)

dataset_path = "/mnt/d/tornetExamples/all_2014"

# Train the model
results_4 = model.train(data="/mnt/d/tornetExamples/all_2014", epochs=100, save=False, save_period=1, batch=4, freeze=7, imgsz=560)

Ultralytics 8.3.36 🚀 Python-3.9.20 torch-2.5.1+cu124 CUDA:0 (NVIDIA GeForce RTX 3070, 8192MiB)
engine/trainer: task=classify, mode=train, model=yolo11n-cls.pt, data=/mnt/d/tornetExamples/all_2014, epochs=100, time=None, patience=100, batch=4, imgsz=560, save=False, save_period=1, cache=False, device=None, workers=8, project=None, name=train13, exist_ok=False, pretrained=True, optimizer=auto, verbose=True, seed=0, deterministic=True, single_cls=False, rect=False, cos_lr=False, close_mosaic=10, resume=False, amp=True, fraction=1.0, profile=False, freeze=7, multi_scale=False, overlap_mask=True, mask_ratio=4, dropout=0.0, val=True, split=val, save_json=False, save_hybrid=False, conf=None, iou=0.7, max_det=300, half=False, dnn=False, plots=True, source=None, vid_stride=1, stream_buffer=False, visualize=False, augment=False, agnostic_nms=False, classes=None, retina_masks=False, embed=None, show=False, save_frames=False, save_txt=False, save_conf=False, save_crop=False, show_labels=True, show

E0000 00:00:1732517775.633792      44 cuda_dnn.cc:8310] Unable to register cuDNN factory: Attempting to register factory for plugin cuDNN when one has already been registered
E0000 00:00:1732517775.648965      44 cuda_blas.cc:1418] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered


Overriding model.yaml nc=80 with nc=2

                   from  n    params  module                                       arguments                     
  0                  -1  1       464  ultralytics.nn.modules.conv.Conv             [3, 16, 3, 2]                 
  1                  -1  1      4672  ultralytics.nn.modules.conv.Conv             [16, 32, 3, 2]                
  2                  -1  1      6640  ultralytics.nn.modules.block.C3k2            [32, 64, 1, False, 0.25]      
  3                  -1  1     36992  ultralytics.nn.modules.conv.Conv             [64, 64, 3, 2]                
  4                  -1  1     26080  ultralytics.nn.modules.block.C3k2            [64, 128, 1, False, 0.25]     
  5                  -1  1    147712  ultralytics.nn.modules.conv.Conv             [128, 128, 3, 2]              
  6                  -1  1     87040  ultralytics.nn.modules.block.C3k2            [128, 128, 1, True]           
  7                  -1  1    295424  ultralytics

train: Scanning /mnt/d/tornetExamples/all_2014/train... 17447 images, 0 corrupt: 100%|██████████| 17447/17447 [00:00<?, 
val: Scanning /mnt/d/tornetExamples/all_2014/test... 2546 images, 0 corrupt: 100%|██████████| 2546/2546 [00:00<?, ?it/s]


optimizer: 'optimizer=auto' found, ignoring 'lr0=0.01' and 'momentum=0.937' and determining best 'optimizer', 'lr0' and 'momentum' automatically... 
optimizer: SGD(lr=0.01, momentum=0.9) with parameter groups 39 weight(decay=0.0), 40 weight(decay=0.0005), 40 bias(decay=0.0)
TensorBoard: model graph visualization added ✅
Image sizes 576 train, 576 val
Using 8 dataloader workers
Logging results to runs/classify/train13
Starting training for 100 epochs...

      Epoch    GPU_mem       loss  Instances       Size


      1/100     0.239G     0.2551          3        576: 100%|██████████| 4362/4362 [09:23<00:00,  7.75it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:50<00:00,  6.32it/s]

                   all      0.861          1



      Epoch    GPU_mem       loss  Instances       Size


      2/100      0.21G     0.2385          3        576: 100%|██████████| 4362/4362 [09:54<00:00,  7.33it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:57<00:00,  5.59it/s]

                   all      0.861          1



      Epoch    GPU_mem       loss  Instances       Size


      3/100     0.222G      0.234          3        576: 100%|██████████| 4362/4362 [10:32<00:00,  6.90it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:53<00:00,  5.97it/s]

                   all      0.862          1



      Epoch    GPU_mem       loss  Instances       Size


      4/100     0.222G     0.2343          3        576: 100%|██████████| 4362/4362 [09:56<00:00,  7.31it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:53<00:00,  5.96it/s]

                   all      0.868          1



      Epoch    GPU_mem       loss  Instances       Size


      5/100     0.222G     0.2286          3        576: 100%|██████████| 4362/4362 [09:49<00:00,  7.41it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:54<00:00,  5.87it/s]

                   all      0.878          1



      Epoch    GPU_mem       loss  Instances       Size


      6/100     0.222G     0.2209          3        576: 100%|██████████| 4362/4362 [09:44<00:00,  7.47it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:52<00:00,  6.06it/s]

                   all      0.864          1



      Epoch    GPU_mem       loss  Instances       Size


      7/100     0.222G     0.2172          3        576: 100%|██████████| 4362/4362 [09:42<00:00,  7.49it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:53<00:00,  5.96it/s]

                   all      0.872          1



      Epoch    GPU_mem       loss  Instances       Size


      8/100     0.222G     0.2127          3        576: 100%|██████████| 4362/4362 [09:44<00:00,  7.46it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:54<00:00,  5.88it/s]

                   all      0.877          1



      Epoch    GPU_mem       loss  Instances       Size


      9/100      0.21G     0.2106          3        576: 100%|██████████| 4362/4362 [09:55<00:00,  7.32it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:54<00:00,  5.89it/s]

                   all      0.871          1



      Epoch    GPU_mem       loss  Instances       Size


     10/100     0.222G     0.2066          3        576: 100%|██████████| 4362/4362 [09:42<00:00,  7.49it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:53<00:00,  5.98it/s]

                   all      0.866          1



      Epoch    GPU_mem       loss  Instances       Size


     11/100     0.222G      0.205          3        576: 100%|██████████| 4362/4362 [09:18<00:00,  7.81it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:53<00:00,  5.96it/s]

                   all      0.871          1



      Epoch    GPU_mem       loss  Instances       Size


     12/100     0.222G     0.2011          3        576: 100%|██████████| 4362/4362 [09:43<00:00,  7.48it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:53<00:00,  5.96it/s]

                   all      0.872          1



      Epoch    GPU_mem       loss  Instances       Size


     13/100     0.222G     0.1952          3        576: 100%|██████████| 4362/4362 [08:55<00:00,  8.15it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:50<00:00,  6.37it/s]

                   all      0.873          1



      Epoch    GPU_mem       loss  Instances       Size


     14/100     0.222G     0.1935          3        576: 100%|██████████| 4362/4362 [09:30<00:00,  7.64it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:53<00:00,  5.97it/s]

                   all      0.871          1



      Epoch    GPU_mem       loss  Instances       Size


     15/100     0.222G     0.1954          3        576: 100%|██████████| 4362/4362 [09:42<00:00,  7.49it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:52<00:00,  6.07it/s]

                   all      0.877          1



      Epoch    GPU_mem       loss  Instances       Size


     16/100     0.222G     0.1881          3        576: 100%|██████████| 4362/4362 [09:47<00:00,  7.42it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:53<00:00,  6.00it/s]

                   all      0.879          1



      Epoch    GPU_mem       loss  Instances       Size


     17/100      0.21G     0.1896          3        576: 100%|██████████| 4362/4362 [09:35<00:00,  7.58it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:53<00:00,  5.97it/s]

                   all      0.875          1



      Epoch    GPU_mem       loss  Instances       Size


     18/100     0.222G     0.1866          3        576: 100%|██████████| 4362/4362 [09:29<00:00,  7.66it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:52<00:00,  6.08it/s]

                   all      0.876          1



      Epoch    GPU_mem       loss  Instances       Size


     19/100     0.222G     0.1847          3        576: 100%|██████████| 4362/4362 [07:39<00:00,  9.50it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.19it/s]

                   all      0.879          1



      Epoch    GPU_mem       loss  Instances       Size


     20/100     0.222G      0.182          3        576: 100%|██████████| 4362/4362 [06:35<00:00, 11.03it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.15it/s]

                   all      0.885          1



      Epoch    GPU_mem       loss  Instances       Size


     21/100     0.222G     0.1849          3        576: 100%|██████████| 4362/4362 [07:35<00:00,  9.57it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.25it/s]

                   all      0.882          1



      Epoch    GPU_mem       loss  Instances       Size


     22/100     0.222G     0.1812          3        576: 100%|██████████| 4362/4362 [06:44<00:00, 10.79it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.25it/s]

                   all      0.881          1



      Epoch    GPU_mem       loss  Instances       Size


     23/100     0.222G     0.1771          3        576: 100%|██████████| 4362/4362 [08:47<00:00,  8.26it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.25it/s]

                   all      0.876          1



      Epoch    GPU_mem       loss  Instances       Size


     24/100     0.222G      0.179          3        576: 100%|██████████| 4362/4362 [07:08<00:00, 10.18it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.14it/s]

                   all      0.876          1



      Epoch    GPU_mem       loss  Instances       Size


     25/100      0.21G     0.1768          3        576: 100%|██████████| 4362/4362 [07:07<00:00, 10.21it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.24it/s]

                   all      0.875          1

      Epoch    GPU_mem       loss  Instances       Size



     26/100     0.222G     0.1778          3        576: 100%|██████████| 4362/4362 [07:04<00:00, 10.28it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:52<00:00,  6.09it/s]

                   all      0.885          1



      Epoch    GPU_mem       loss  Instances       Size


     27/100     0.222G     0.1741          3        576: 100%|██████████| 4362/4362 [07:04<00:00, 10.28it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:52<00:00,  6.11it/s]

                   all       0.88          1



      Epoch    GPU_mem       loss  Instances       Size


     28/100     0.222G     0.1743          3        576: 100%|██████████| 4362/4362 [07:20<00:00,  9.90it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.16it/s]

                   all      0.881          1



      Epoch    GPU_mem       loss  Instances       Size


     29/100     0.222G     0.1681          3        576: 100%|██████████| 4362/4362 [07:40<00:00,  9.47it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:50<00:00,  6.33it/s]

                   all       0.88          1



      Epoch    GPU_mem       loss  Instances       Size


     30/100     0.222G     0.1753          3        576: 100%|██████████| 4362/4362 [08:45<00:00,  8.30it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.15it/s]

                   all      0.878          1



      Epoch    GPU_mem       loss  Instances       Size


     31/100     0.222G     0.1676          3        576: 100%|██████████| 4362/4362 [07:05<00:00, 10.26it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.17it/s]

                   all      0.877          1



      Epoch    GPU_mem       loss  Instances       Size


     32/100     0.222G     0.1667          3        576: 100%|██████████| 4362/4362 [07:06<00:00, 10.22it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.14it/s]

                   all      0.886          1



      Epoch    GPU_mem       loss  Instances       Size


     33/100      0.21G     0.1669          3        576: 100%|██████████| 4362/4362 [07:34<00:00,  9.61it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.22it/s]

                   all      0.883          1

      Epoch    GPU_mem       loss  Instances       Size



     34/100     0.222G     0.1642          3        576: 100%|██████████| 4362/4362 [07:24<00:00,  9.81it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.21it/s]

                   all      0.886          1



      Epoch    GPU_mem       loss  Instances       Size


     35/100     0.222G     0.1605          3        576: 100%|██████████| 4362/4362 [07:22<00:00,  9.87it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:50<00:00,  6.28it/s]

                   all      0.888          1



      Epoch    GPU_mem       loss  Instances       Size


     36/100     0.222G      0.163          3        576: 100%|██████████| 4362/4362 [07:43<00:00,  9.41it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.15it/s]

                   all      0.885          1



      Epoch    GPU_mem       loss  Instances       Size


     37/100     0.222G     0.1604          3        576: 100%|██████████| 4362/4362 [07:27<00:00,  9.75it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:52<00:00,  6.09it/s]

                   all      0.882          1



      Epoch    GPU_mem       loss  Instances       Size


     38/100     0.222G     0.1586          3        576: 100%|██████████| 4362/4362 [07:46<00:00,  9.36it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.15it/s]

                   all      0.886          1



      Epoch    GPU_mem       loss  Instances       Size


     39/100     0.222G     0.1586          3        576: 100%|██████████| 4362/4362 [07:23<00:00,  9.84it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.18it/s]

                   all      0.885          1



      Epoch    GPU_mem       loss  Instances       Size


     40/100     0.222G     0.1595          3        576: 100%|██████████| 4362/4362 [07:21<00:00,  9.89it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:50<00:00,  6.26it/s]

                   all      0.885          1



      Epoch    GPU_mem       loss  Instances       Size


     41/100      0.21G     0.1565          3        576: 100%|██████████| 4362/4362 [07:25<00:00,  9.78it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:50<00:00,  6.26it/s]

                   all      0.885          1



      Epoch    GPU_mem       loss  Instances       Size


     42/100     0.222G     0.1539          3        576: 100%|██████████| 4362/4362 [07:20<00:00,  9.91it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.20it/s]

                   all      0.884          1



      Epoch    GPU_mem       loss  Instances       Size


     43/100     0.222G     0.1545          3        576: 100%|██████████| 4362/4362 [08:01<00:00,  9.05it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.16it/s]

                   all      0.885          1



      Epoch    GPU_mem       loss  Instances       Size


     44/100     0.222G     0.1504          3        576: 100%|██████████| 4362/4362 [07:25<00:00,  9.79it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.14it/s]

                   all      0.887          1



      Epoch    GPU_mem       loss  Instances       Size


     45/100     0.222G     0.1523          3        576: 100%|██████████| 4362/4362 [08:17<00:00,  8.77it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.21it/s]

                   all      0.886          1



      Epoch    GPU_mem       loss  Instances       Size


     46/100     0.222G     0.1486          3        576: 100%|██████████| 4362/4362 [07:30<00:00,  9.69it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:52<00:00,  6.06it/s]

                   all      0.886          1



      Epoch    GPU_mem       loss  Instances       Size


     47/100     0.222G     0.1525          3        576: 100%|██████████| 4362/4362 [07:27<00:00,  9.74it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.15it/s]

                   all      0.886          1



      Epoch    GPU_mem       loss  Instances       Size


     48/100     0.222G     0.1515          3        576: 100%|██████████| 4362/4362 [07:27<00:00,  9.75it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:50<00:00,  6.30it/s]

                   all      0.885          1



      Epoch    GPU_mem       loss  Instances       Size


     49/100      0.21G     0.1454          3        576: 100%|██████████| 4362/4362 [07:34<00:00,  9.60it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.17it/s]

                   all      0.885          1



      Epoch    GPU_mem       loss  Instances       Size


     50/100     0.222G     0.1496          3        576: 100%|██████████| 4362/4362 [08:16<00:00,  8.79it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:52<00:00,  6.07it/s]

                   all      0.886          1



      Epoch    GPU_mem       loss  Instances       Size


     51/100     0.222G     0.1444          3        576: 100%|██████████| 4362/4362 [07:37<00:00,  9.53it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.16it/s]

                   all      0.886          1



      Epoch    GPU_mem       loss  Instances       Size


     52/100     0.222G     0.1436          3        576: 100%|██████████| 4362/4362 [08:10<00:00,  8.89it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.25it/s]

                   all      0.887          1



      Epoch    GPU_mem       loss  Instances       Size


     53/100     0.222G     0.1445          3        576: 100%|██████████| 4362/4362 [07:37<00:00,  9.53it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.22it/s]

                   all      0.888          1



      Epoch    GPU_mem       loss  Instances       Size


     54/100     0.222G     0.1417          3        576: 100%|██████████| 4362/4362 [07:42<00:00,  9.44it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.25it/s]

                   all      0.888          1



      Epoch    GPU_mem       loss  Instances       Size


     55/100     0.222G       0.14          3        576: 100%|██████████| 4362/4362 [07:45<00:00,  9.38it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.14it/s]

                   all      0.888          1



      Epoch    GPU_mem       loss  Instances       Size


     56/100     0.222G     0.1411          3        576: 100%|██████████| 4362/4362 [07:32<00:00,  9.64it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:52<00:00,  6.09it/s]

                   all      0.886          1



      Epoch    GPU_mem       loss  Instances       Size


     57/100      0.21G     0.1377          3        576: 100%|██████████| 4362/4362 [08:18<00:00,  8.75it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:52<00:00,  6.10it/s]

                   all      0.886          1



      Epoch    GPU_mem       loss  Instances       Size


     58/100     0.222G     0.1383          3        576: 100%|██████████| 4362/4362 [07:42<00:00,  9.43it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:52<00:00,  6.10it/s]

                   all      0.886          1



      Epoch    GPU_mem       loss  Instances       Size


     59/100     0.222G     0.1368          3        576: 100%|██████████| 4362/4362 [08:01<00:00,  9.05it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:52<00:00,  6.11it/s]

                   all      0.888          1



      Epoch    GPU_mem       loss  Instances       Size


     60/100     0.222G     0.1334          3        576: 100%|██████████| 4362/4362 [07:46<00:00,  9.34it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:52<00:00,  6.02it/s]

                   all      0.889          1



      Epoch    GPU_mem       loss  Instances       Size


     61/100     0.222G     0.1339          3        576: 100%|██████████| 4362/4362 [07:53<00:00,  9.21it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.24it/s]

                   all       0.89          1



      Epoch    GPU_mem       loss  Instances       Size


     62/100     0.222G     0.1344          3        576: 100%|██████████| 4362/4362 [07:45<00:00,  9.36it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.22it/s]

                   all      0.891          1



      Epoch    GPU_mem       loss  Instances       Size


     63/100     0.222G     0.1315          3        576: 100%|██████████| 4362/4362 [07:39<00:00,  9.49it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.21it/s]

                   all      0.892          1



      Epoch    GPU_mem       loss  Instances       Size


     64/100     0.222G     0.1317          3        576: 100%|██████████| 4362/4362 [08:09<00:00,  8.92it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:52<00:00,  6.05it/s]

                   all      0.892          1



      Epoch    GPU_mem       loss  Instances       Size


     65/100      0.21G     0.1307          3        576: 100%|██████████| 4362/4362 [08:32<00:00,  8.51it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:52<00:00,  6.12it/s]

                   all      0.892          1



      Epoch    GPU_mem       loss  Instances       Size


     66/100     0.222G     0.1276          3        576: 100%|██████████| 4362/4362 [07:48<00:00,  9.31it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.17it/s]

                   all      0.892          1



      Epoch    GPU_mem       loss  Instances       Size


     67/100     0.222G     0.1281          3        576: 100%|██████████| 4362/4362 [07:32<00:00,  9.65it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.22it/s]

                   all      0.891          1



      Epoch    GPU_mem       loss  Instances       Size


     68/100     0.222G     0.1246          3        576: 100%|██████████| 4362/4362 [07:39<00:00,  9.49it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.25it/s]

                   all      0.892          1



      Epoch    GPU_mem       loss  Instances       Size


     69/100     0.222G     0.1276          3        576: 100%|██████████| 4362/4362 [08:12<00:00,  8.86it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.19it/s]

                   all      0.892          1



      Epoch    GPU_mem       loss  Instances       Size


     70/100     0.222G     0.1243          3        576: 100%|██████████| 4362/4362 [07:54<00:00,  9.19it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:50<00:00,  6.26it/s]

                   all      0.892          1



      Epoch    GPU_mem       loss  Instances       Size


     71/100     0.222G     0.1191          3        576: 100%|██████████| 4362/4362 [07:57<00:00,  9.13it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.15it/s]

                   all      0.891          1



      Epoch    GPU_mem       loss  Instances       Size


     72/100     0.222G     0.1218          3        576: 100%|██████████| 4362/4362 [09:06<00:00,  7.98it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:52<00:00,  6.13it/s]

                   all       0.89          1



      Epoch    GPU_mem       loss  Instances       Size


     73/100      0.21G     0.1214          3        576: 100%|██████████| 4362/4362 [07:38<00:00,  9.51it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:52<00:00,  6.13it/s]

                   all       0.89          1

      Epoch    GPU_mem       loss  Instances       Size



     74/100     0.222G     0.1224          3        576: 100%|██████████| 4362/4362 [07:44<00:00,  9.39it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.21it/s]

                   all      0.889          1



      Epoch    GPU_mem       loss  Instances       Size


     75/100     0.222G      0.119          3        576: 100%|██████████| 4362/4362 [07:32<00:00,  9.64it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:50<00:00,  6.26it/s]

                   all      0.889          1



      Epoch    GPU_mem       loss  Instances       Size


     76/100     0.222G     0.1136          3        576: 100%|██████████| 4362/4362 [07:35<00:00,  9.58it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:52<00:00,  6.10it/s]

                   all       0.89          1



      Epoch    GPU_mem       loss  Instances       Size


     77/100     0.222G     0.1159          3        576: 100%|██████████| 4362/4362 [07:52<00:00,  9.24it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.16it/s]

                   all      0.889          1



      Epoch    GPU_mem       loss  Instances       Size


     78/100     0.222G     0.1154          3        576: 100%|██████████| 4362/4362 [07:40<00:00,  9.48it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:53<00:00,  6.01it/s]

                   all      0.888          1



      Epoch    GPU_mem       loss  Instances       Size


     79/100     0.222G     0.1131          3        576: 100%|██████████| 4362/4362 [09:13<00:00,  7.88it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.14it/s]

                   all      0.889          1



      Epoch    GPU_mem       loss  Instances       Size


     80/100     0.222G      0.107          3        576: 100%|██████████| 4362/4362 [07:24<00:00,  9.80it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.18it/s]

                   all       0.89          1



      Epoch    GPU_mem       loss  Instances       Size


     81/100      0.21G      0.109          3        576: 100%|██████████| 4362/4362 [07:40<00:00,  9.47it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.25it/s]

                   all      0.889          1



      Epoch    GPU_mem       loss  Instances       Size


     82/100     0.222G     0.1098          3        576: 100%|██████████| 4362/4362 [07:34<00:00,  9.60it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:52<00:00,  6.10it/s]

                   all      0.889          1



      Epoch    GPU_mem       loss  Instances       Size


     83/100     0.222G     0.1091          3        576: 100%|██████████| 4362/4362 [07:29<00:00,  9.71it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:52<00:00,  6.06it/s]

                   all      0.889          1



      Epoch    GPU_mem       loss  Instances       Size


     84/100     0.222G     0.1054          3        576: 100%|██████████| 4362/4362 [08:16<00:00,  8.79it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:52<00:00,  6.13it/s]

                   all      0.889          1



      Epoch    GPU_mem       loss  Instances       Size


     85/100     0.222G     0.1082          3        576: 100%|██████████| 4362/4362 [07:28<00:00,  9.72it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:52<00:00,  6.12it/s]

                   all      0.889          1



      Epoch    GPU_mem       loss  Instances       Size


     86/100     0.222G     0.1078          3        576: 100%|██████████| 4362/4362 [08:55<00:00,  8.14it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.16it/s]

                   all      0.888          1



      Epoch    GPU_mem       loss  Instances       Size


     87/100     0.222G     0.1043          3        576: 100%|██████████| 4362/4362 [07:34<00:00,  9.59it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.16it/s]

                   all      0.889          1



      Epoch    GPU_mem       loss  Instances       Size


     88/100     0.222G     0.1035          3        576: 100%|██████████| 4362/4362 [07:48<00:00,  9.31it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:52<00:00,  6.12it/s]

                   all      0.889          1



      Epoch    GPU_mem       loss  Instances       Size


     89/100      0.21G    0.09699          3        576: 100%|██████████| 4362/4362 [07:46<00:00,  9.36it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.15it/s]

                   all      0.888          1



      Epoch    GPU_mem       loss  Instances       Size


     90/100     0.222G     0.1044          3        576: 100%|██████████| 4362/4362 [07:43<00:00,  9.41it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.15it/s]

                   all      0.888          1



      Epoch    GPU_mem       loss  Instances       Size


     91/100     0.239G     0.1013          3        576: 100%|██████████| 4362/4362 [08:31<00:00,  8.53it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:52<00:00,  6.04it/s]

                   all      0.888          1



      Epoch    GPU_mem       loss  Instances       Size


     92/100     0.222G    0.09459          3        576: 100%|██████████| 4362/4362 [09:03<00:00,  8.02it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:52<00:00,  6.07it/s]

                   all      0.888          1



      Epoch    GPU_mem       loss  Instances       Size


     93/100     0.222G    0.09375          3        576: 100%|██████████| 4362/4362 [08:55<00:00,  8.14it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.15it/s]

                   all      0.887          1



      Epoch    GPU_mem       loss  Instances       Size


     94/100     0.222G    0.09414          3        576: 100%|██████████| 4362/4362 [08:55<00:00,  8.15it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:51<00:00,  6.14it/s]

                   all      0.886          1



      Epoch    GPU_mem       loss  Instances       Size


     95/100     0.222G    0.09383          3        576: 100%|██████████| 4362/4362 [08:11<00:00,  8.87it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:53<00:00,  6.01it/s]

                   all      0.886          1



      Epoch    GPU_mem       loss  Instances       Size


     96/100     0.222G    0.09552          3        576: 100%|██████████| 4362/4362 [07:58<00:00,  9.12it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:46<00:00,  6.92it/s]

                   all      0.886          1



      Epoch    GPU_mem       loss  Instances       Size


     97/100      0.21G     0.0926          3        576: 100%|██████████| 4362/4362 [07:29<00:00,  9.71it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:52<00:00,  6.07it/s]

                   all      0.886          1



      Epoch    GPU_mem       loss  Instances       Size


     98/100     0.222G    0.08851          3        576: 100%|██████████| 4362/4362 [08:14<00:00,  8.83it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:53<00:00,  5.91it/s]

                   all      0.886          1



      Epoch    GPU_mem       loss  Instances       Size


     99/100     0.222G    0.08966          3        576: 100%|██████████| 4362/4362 [09:45<00:00,  7.45it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:52<00:00,  6.10it/s]

                   all      0.886          1



      Epoch    GPU_mem       loss  Instances       Size


    100/100     0.222G    0.08937          3        576: 100%|██████████| 4362/4362 [09:15<00:00,  7.85it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 319/319 [00:52<00:00,  6.11it/s]

                   all      0.886          1



100 epochs completed in 15.067 hours.
Optimizer stripped from runs/classify/train13/weights/last.pt, 3.2MB


# Results

Our best accuracy (not even AUC, but ACCURACY) after 100 epochs here was only .892 - that's pretty poor.

For comparison, both our improved model using flipped tornadoes to augment the dataset and the initial TorNet CNN both had accuracies that were much higher, well above .94 in both cases.

Let's try some other hyperparameter settings in future notebooks to see if we can do any better.